flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/integration/test/paimon/FlinkPaimonCatalogIT.java

```diff
@@ -47,11 +47,6 @@ protected boolean supportColumnOperation() {
     return false;
   }
 
-  @Override
```

Comment:
could we remove `supportColumnOperations` too?

Reply:
done